internal/events/dx_callbacks.go

```diff
@@ -64,51 +64,49 @@ func (em *eventManager) MessageReceived(dx dataexchange.Plugin, peerID string, d
 	return string(manifestBytes), err
 }
 
-func (em *eventManager) checkReceivedIdentity(ctx context.Context, peerID, author, signingKey string) (node *fftypes.Node, err error) {
+// the data came from) matches the org listed in the batch. The on-chain identity check
+// is performed by the aggregator, across broadcast and private consistently.
+func (em *eventManager) checkReceivedOffchainIdentity(ctx context.Context, peerID, author string) (node *fftypes.Identity, err error) {
```

Comment:
Renamed to make it clear that this only verifies the off-chain identity at this point.